crates/aptos-faucet/metrics-server/src/config.rs

```diff
@@ -18,6 +18,16 @@ pub struct MetricsServerConfig {
     pub listen_port: u16,
 }
 
+impl Default for MetricsServerConfig {
```

Comment:
I feel that aptos-faucet becomes the unnecessary dependency for other projects tho only the metrics part i needed. 

should we abstract a common metrics library?